docs/reference/objects/product/variant/index.md

```diff
@@ -15,7 +15,8 @@ object
 [Experience Variant]({% link docs/reference/objects/product/variant/experience_variant/index.md %}) and
 [Accommodation Variant]({% link docs/reference/objects/product/variant/accommodation_variant/index.md %}).
 
-<br>
+
+The variant object will behave slightly differently depending on how it is accessed. If it is accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), some methods will return a result that is specific to that given date. If it is access independently or through a product, then those methods will take into account all dates on the product. Please check the method descriptions for more details.
```

Comment:
**nitpick** I would turn these last two points into bullet points here to improve readability, just a bit quick to digest. It also seems to follow the pattern of how the other sections are written. As in:
```
The variant object will behave slightly differently depending on how it is accessed. 
- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), some methods will return a result that is specific to that given date. 
- When accessed independently or through a product, these methods will take into account all dates on the product.
Please check the method descriptions for more details.
```